packages/blade/src/components/VisuallyHidden/VisuallyHidden.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import {
+  Title as StorybookTitle,
+  Subtitle,
+  Primary,
+  ArgsTable,
+  Stories,
+  PRIMARY_STORY,
+} from '@storybook/addon-docs';
+import { Highlight, Link } from '@storybook/design-system';
+import type { ReactElement } from 'react';
+import { Text } from '../Typography';
+import type { VisuallyHiddenProps } from './VisuallyHidden';
+import VisuallyHiddenComponent from './VisuallyHidden';
+
+const Page = (): ReactElement => {
+  return (
+    <>
+      <StorybookTitle />
+      <Subtitle>
+        The VisuallyHidden component makes content hidden from sighted users but available for
+        screen reader users.
+      </Subtitle>
+      <Link
+        withArrow={true}
+        href="https://github.com/razorpay/blade/blob/anu/a11y-rfc/rfcs/2022-04-09-accessibility.md#hidden-content"
+        target="_blank"
+      >
+        See Hidden Content RFC
+      </Link>
+      <br />
+      <br />
+      <StorybookTitle>Usage</StorybookTitle>
+      <Highlight language="tsx">{`import { VisuallyHidden } from '@razorpay/blade/components' \nimport type { VisuallyHiddenProps } from '@razorpay/blade/components'`}</Highlight>
+      <StorybookTitle>Example</StorybookTitle>
+      <Primary />
+      <StorybookTitle>Properties</StorybookTitle>
+      <ArgsTable story={PRIMARY_STORY} />
```

Comment:
If there are no props, lets remove this section?
```suggestion
```

Reply:
Actually it does accept `children` as a prop. Why is that not showing up in the story?

Reply:
Because I haven't passed the children in.